roles/static_inventory/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Either to checkpoint the dynamic inventory into a static one
+refresh_inventory: True
+inventory: static
+inventory_dir: ~/openstack-inventory
```

Comment:
Oh, right. I forgot this is just the role. Right, so we may want to remove `openstack_inventory_dir` from that PR and leave this unchanged.

Let me try that and see.